assets/scss/tocbot.css

```
@@ -0,0 +1,31 @@
+.toc{overflow-y:auto}.toc>.toc-list{overflow:hidden;position:relative}.toc>.toc-list li{list-style:none}.toc-list{margin:0;padding-left:10px}a.toc-link{color:currentColor;height:100%;word-wrap:break-word;white-space: normal;}.is-collapsible{max-height:100000px;overflow:hidden;transition:all 300ms ease-in-out}.is-collapsed{max-height:0}.is-position-fixed{position:fixed !important;top:0}.is-active-link{font-weight:700}.toc-link::before{background-color:#EEE;content:' ';display:inline-block;height:inherit;left:0;margin-top:-1px;position:absolute;width:2px}.is-active-link::before{background-color:$primary;}
```

CSS specifies style for 'toc' but the class assigned to the ToC div is 'td-toc'.  Changing this in my setup fixes height inheritance issues that were resulting in excessive dead space below the last entry in the ToC.

```suggestion
.td-toc{overflow-y:auto}.td-toc>.toc-list{overflow:hidden;position:relative}.td-toc>.toc-list li{list-style:none}.toc-list{margin:0;padding-left:10px}a.toc-link{color:currentColor;height:100%;word-wrap:break-word;white-space: normal;}.is-collapsible{max-height:100000px;overflow:hidden;transition:all 300ms ease-in-out}.is-collapsed{max-height:0}.is-position-fixed{position:fixed !important;top:0}.is-active-link{font-weight:700}.toc-link::before{background-color:#EEE;content:' ';display:inline-block;height:inherit;left:0;margin-top:-1px;position:absolute;width:2px}.is-active-link::before{background-color:$primary;}
```

---

Thanks a lot!